src/AssetsTransferApi.spec.ts

```diff
@@ -22,6 +23,11 @@ const systemAssetsApi = new AssetsTransferApi(
 	2
 );
 const relayAssetsApi = new AssetsTransferApi(adjustedMockRelayApi, 'kusama', 2);
+const moonbeamAssetsApi = new AssetsTransferApi(
```

Comment:
we intentionally name this moonbeam rather than moonriver?

Reply:
Nice I switched it all to moonriver. Nice catch

Reply:
fixed